feg/gateway/services/aaa/servicers/accounting.go

```diff
@@ -333,7 +333,7 @@ func createSessionOnSessionManager(mac net.HardwareAddr, subscriberId *lte_proto
 		RatSpecificContext: &lte_protos.RatSpecificContext{
 			Context: &lte_protos.RatSpecificContext_WlanContext{
 				WlanContext: &lte_protos.WLANSessionContext{
-					HardwareAddr:    mac,
+					MacAddrBinary:   mac,
```

Comment:
just curios, why is the mac address a binary?

Reply:
I'm not exactly sure on the details, but it looks like when it is passed up to SessionProxy, it expects a bytes value.

Reply:
Probably leak of AVP then